src/SonarQube.Bootstrapper/Program.cs

```diff
@@ -89,5 +102,30 @@ private static ILegacyTeamBuildFactory GetLegacyTeamBuildFactory()
             return new NotSupportedLegacyTeamBuildFactory();
 #endif
         }
+
+        [Conditional("DEBUG")]
+        private static void DumpLoadedAssemblies(ILogger logger)
```

Comment:
Can we include the runtime version of the framework?

Reply:
There should be enough info to work that out e.g.

> System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089,C:\WINDOWS\Microsoft.Net\assembly\GAC_MSIL\System.Core\v4.0_4.0.0.0__b77a5c561934e089\System.Core.dll

> System.Xml.XmlSerializer, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a,C:\Program Files\dotnet\shared\Microsoft.NETCore.App\2.0.5\System.Xml.XmlSerializer.dll